applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc

```diff
@@ -5,93 +5,22 @@ include::_attributes/common-attributes.adoc[]
 include::_attributes/servicebinding-document-attributes.adoc[]
 :context: odc-connecting-an-application-to-a-service-using-the-developer-perspective
 
+toc::[]
+
 [role="_abstract"]
 In addition to grouping multiple components within an application, you can also use the *Topology* view to connect components with each other. You can either use a binding connector or a visual one to connect components.
 
-A binding connection between the components can be established only if the target node is an Operator-backed service. This is indicated by the *Create a binding connector* tool-tip which appears when you drag an arrow to such a target node. When an application is connected to a service using a binding connector a `ServiceBinding` resource is created. Then, the Service Binding Operator controller projects the necessary binding data into the application deployment. After the request is successful, the application is redeployed establishing an interaction between the connected components.
+A binding connection between the components can be established only if the target node is an Operator-backed service. This is indicated by the *Create a binding connector* tool-tip which appears when you drag an arrow to such a target node. When an application is connected to a service by using a binding connector a `ServiceBinding` resource is created. Then, the Service Binding Operator controller projects the necessary binding data into the application deployment. After the request is successful, the application is redeployed establishing an interaction between the connected components.
```

Comment:
1. Add a comma before "which"
2. Add a comma after "binding connector" Also, this sentence uses two be verbs, which is passive. If you know the subject of the action, you could make the sentence a bit stronger. In the first part of the sentence, you could say, "When an application uses a binding connector to connect to a service, ..." Then, "<something> creates a ServiceBinding resource. This eliminates the passive voice.

Reply:
Apologies John, commits from other writers (including this file) got included in my PR. Therefore, I do not include this feedback.